### PR TITLE
Result page

### PR DESCRIPTION
**Pull request by dipen2000**
I created this result page for the quiz app so please review it and give me your feedback.

### CODE REVIEW

**Review by pnchinmay — APPROVED** [review]

**Comment by shruthipatil11** [comment]
Looks great.

File: CSS/styles.css [hunk 1]
```diff
@@ -77,17 +116,83 @@ ol{
 }
 
 .rules-container{
-    border : 3px solid yellow;
+    /* border : 3px solid yellow; */
```
Comment (dev-ashishsethi) [inline CSS/styles.css]: avoid commenting code

File: CSS/styles.css [hunk 2]
```diff
@@ -77,17 +116,83 @@ ol{
...
+}
 
+.single-category-container{
+    /* border : 3px solid green; */
+    display : flex;
+    flex-direction: column;
+    padding : 1.2rem;
+    width : 35rem;
+    gap : 1rem;
+    border-radius: 0.8rem;
```
Comment (dev-ashishsethi) [inline CSS/styles.css:142]: use pixels

**Review by dev-ashishsethi — APPROVED** [review]

**Review by j24m — APPROVED** [review]
Everything looks pretty much fine to me , the code and the UI , just make the changes that have been repeatedly mentioned , but yes nice and clean work there :), the margin and padding everywhere in the UI is consistent and nothing looks out of place. :)

**Comment by shruthipatil11** [comment]
looks good to merge

File: CSS/styles.css [hunk 3]
```diff
@@ -77,17 +116,83 @@ ol{
...
+}
 
+.question-container{
+    /* border : 3px solid blue; */
+    width : 60%;
+    max-width: 60rem;
+    padding : 1rem;
+    border-radius: 1rem;
+    background-color : var(--light-test-color);
+}
```
Comment (rahulyadav139) [inline CSS/styles.css:168]: The files are readable and color variable naming is quite simple and understandable. Great work.

**Review by rahulyadav139 — APPROVED** [review]

**Review by j24m — APPROVED** [review]

**Review by j24m — APPROVED** [review]
I see that you have implemented dark mode , looks good, good work : )

**Pull request merged**